deploy/charts/trust-manager/templates/trust.cert-manager.io_bundles.yaml

```diff
@@ -153,32 +153,46 @@ spec:
                   description: List of status conditions to indicate the status of the Bundle. Known condition types are `Bundle`.
                   type: array
                   items:
-                    description: BundleCondition contains condition information for a Bundle.
+                    description: "Condition contains details for one aspect of the current state of this API Resource. --- This struct is intended for direct use as an array at the field path .status.conditions.  For example, \n type FooStatus struct{ // Represents the observations of a foo's current state. // Known .status.conditions.type are: \"Available\", \"Progressing\", and \"Degraded\" // +patchMergeKey=type // +patchStrategy=merge // +listType=map // +listMapKey=type Conditions []metav1.Condition `json:\"conditions,omitempty\" patchStrategy:\"merge\" patchMergeKey:\"type\" protobuf:\"bytes,1,rep,name=conditions\"` \n // other fields }"
```

Comment:
We should find a way to fix this description.

Reply:
Yes, I will look into that now.

Reply:
> I could ask on sig-api-machinery Slack
That would be awesome! Also, maybe we can learn from other projects that do this?

Reply:
I think most projects just take the easy way out and lives with the problem. If you do a quick search in indexed GH projects, there are many hits for this "nonsense": https://github.com/search?q=%22Represents+the+observations+of+a+foo%27s+current+state.%22&type=code

Reply:
Asked a question on [Slack](https://kubernetes.slack.com/archives/C0EG7JC6T/p1700754470441259).